src/components/sidebar/FilterSidebar.jsx

```diff
@@ -28,13 +71,15 @@ const FilterSidebar = () => {
         <div className={classes['filter-type']}>
           <h2 className={classes['filter-type-title']}>Ratings</h2>
```

Comment:
in line 16 of ProductCard.jsx you are using the `classes.className` convention of module CSS. But here you are using `classes[className]` convention. I would suggest ~~to be~~ being consistent